src/content/Avatar/index.jsx

```diff
@@ -1,11 +1,10 @@
 import React, { lazy, Component, Suspense } from "react";
 import Playground from "./avatar.playground.jsx";
-import { importMDX } from "mdx.macro";
+import importMDX from "mdx.macro";
 import PropDrawer from "../../components/PropDrawer";
 
 const Content = lazy(() => importMDX("./avatar.mdx"));
-class App extends Component {
-  render() {
+export default function App(){
```

Comment:
Can you convert this to a named function. Default export of anonymous funcs is not recommended 
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-anonymous-default-export.md

Reply:
sure